backend/src/Squidex/Areas/Api/Controllers/Contents/Models/ContentDto.cs

```diff
@@ -184,6 +184,11 @@ private ContentDto CreateLinksAsync(IEnrichedContentEntity content, Resources re
                     AddPutLink("update", resources.Url<ContentsController>(x => nameof(x.PutContent), values));
                 }
 
+                if (resources.CanUpdateContentOwn(schema))
```

Comment:
The resources is a dictionary, when you add 2 resources with the same name "update" you are overriding the previous value.

Reply:
Maybe something like this?

```
if (resources.CanUpdateContent(schema) || resources.CanUpdateContentOwn(schema))
                {
                    AddPutLink("update", resources.Url<ContentsController>(x => nameof(x.PutContent), values));
                }
```

Reply:
You only need to check for .own